SUBMITPR.md

```diff
@@ -2,6 +2,11 @@
 
 Pull requests let you tell others about changes you've pushed to a repository on GitHub. Once a pull request is created, interested parties can review the set of changes, discuss potential modifications, and even push follow-up commits if necessary. Anyone can submit a pull request and they're useful in the shared repository model where they're used to initiate code review and general discussion about a set of changes before being merged into a mainline branch.
 
+ **Make sure that you follow the guidelines below**
+ - **The intent is to upload simple apps created by esri staff**
```

Comment:
How about something like this:

###Guidelines for Esri Staff
* Do **not** upload code provided by a user unless you have received explicit permission.**
* If a user is looking for a specific sample that does not exist in our resource center and you are able to create it, then you can upload this sample. However, this sample should not include any names, comments, namespaces that are related with the user.

I don't really like adding an explicit statement that apps need to be created by esri staff, because i think we should be encouraging independent contributions from our user community as well.